test/mountain_view/component_test.rb

```diff
@@ -16,16 +16,112 @@ def test_humanized_title
   def test_styleguide_stubs
     component = MountainView::Component.new("header")
 
-    expected_stub = {
-      id: 1,
-      title: "20 Mountains you didn't know they even existed",
-      subtitle: "Buzzfeed title"
-    }
+    expected_stub =
+      {
+        :meta => "There is this different classes",
+        :stubs =>
+          [
+            { :id  =>  1, :title => "20 Mountains you didn't know they even existed",
```

Comment:
Line is too long. [85/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.